<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="guide_link_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>User guide</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_input_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>File input</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filename_input">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="A_input_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>To enter A click here</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_input_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>171</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>To enter b click here</string>
    </property>
   </widget>
   <widget class="QPushButton" name="solve_button">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>solve</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>20</width>
      <height>261</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>130</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>241</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>pig</string>
    </property>
   </widget>
   <widget class="QLabel" name="accuracy_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>155</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Accuracy</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="accuracy_input">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="criterion_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Criterion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="criterion_input">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="history_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>781</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="load_button">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>60</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>load</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
